lib/logstash/plugin_mixins/opensearch/api_configs.rb

```diff
@@ -1,22 +1,31 @@
-module LogStash; module PluginMixins; module ElasticSearch
+# SPDX-License-Identifier: Apache-2.0
+#
+#  The OpenSearch Contributors require contributions made to
+#  this file be licensed under the Apache-2.0 license or a
+#  compatible open source license.
+#
+#  Modifications Copyright OpenSearch Contributors. See
+#  GitHub history for details.
+
+module LogStash; module PluginMixins; module OpenSearch
   module APIConfigs
 
-    # This module defines common options that can be reused by alternate elasticsearch output plugins such as the elasticsearch_data_streams output.
+    # This module defines common options that can be reused by alternate OpenSearch output plugins such as the OpenSearch_data_streams output.
```

Comment:
I believe data streams are xpack feature. Can this be removed?

Reply:
Ack